MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+## Project Maintainers
```

Comment:
This file seems to duplicate the content in our README, see https://github.com/strongloop/loopback-next#team

@dhmlau Please find a way how to have a single place listing the maintainers.